Darwin/Foundation-swiftoverlay/Date.swift

```diff
@@ -19,7 +19,7 @@ import CoreFoundation
  
  A `Date` is independent of a particular calendar or time zone. To represent a `Date` to a user, you must interpret it in the context of a `Calendar`.
 */
-public struct Date : ReferenceConvertible, Comparable, Equatable {
+public struct Date : ReferenceConvertible, Comparable, Equatable, @unchecked Sendable {
```

Comment:
Does this need to be `@unchecked`? I think this can just be `Sendable` since the type is fairly trivial, and that way we can have some compiler validation on the conformance